### ISSUE TITLE
RimPy Strip Mods error on Linux, and can't access internet

### ISSUE THREAD

**Issue by compulshun** [rimpy-custom/RimPy#93]
Here's what I get from my console - the last two lines are after running 'strip mods'. The NSS errors happened as a result of pressing the "RimPy Wiki" button:

└─$ ./RimPy
Ошибка открытия URL: https://api.github.com/repos/rimpy-custom/RimPy/releases
Sandboxing disabled by user.
Ошибка открытия URL: https://content-system.gog.com/products/1094900565/os/windows/builds?generation=2
Ошибка открытия URL: https://rimworldwiki.com/wiki/Version/Current_version
9.213049708000199
[171828:171849:1004/130340.332150:ERROR:nss_util.cc(166)] Error initializing NSS with a persistent database (sql:/home/default/.pki/nssdb): libsoftokn3.so: cannot open shared object file: No such file or directory
[171828:171849:1004/130340.332239:ERROR:nss_util.cc(124)] Error initializing NSS without a persistent database: NSS error code: -5925
[171828:171849:1004/130340.332250:FATAL:nss_util.cc(126)] nss_error=-5925, os_error=0
[171780:171780:1004/130340.533705:ERROR:network_service_instance_impl.cc(262)] Network service crashed, restarting service.
js: Uncaught TypeError: Cannot read property 'lastChild' of undefined
js: Uncaught TypeError: Cannot read property 'parentElement' of null
kde-open5: symbol lookup error: /lib/x86_64-linux-gnu/libKF5GuiAddons.so.5: undefined symbol: wl_proxy_marshal_flags
QCoreApplication::exec: The event loop is already running
[1]    171780 segmentation fault (core dumped)  ./RimPy

Let me know if there's more information I can give you or a way to get a more detailed log file.

**Comment by rimpy-custom** [comment]
Try latest prerelease, let me know if issue wasn't fixed

**Comment by erkoiv** [comment]
Linux 20.04 does not have a high enough libc6 version. 

```
./RimPy: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./RimPy)
./RimPy: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.35' not found (required by /home/username/Desktop/RimPy/RimPy_Linux/libpython3.10.so.1.0)
./RimPy: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /home/username/Desktop/RimPy/RimPy_Linux/libpython3.10.so.1.0)
./RimPy: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /home/username/Desktop/RimPy/RimPy_Linux/libpython3.10.so.1.0)
./RimPy: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /home/username/Desktop/RimPy/RimPy_Linux/libpython3.10.so.1.0)
```

My fully updated 20.04 LTS system is running libc6 2.31.

**Comment by compulshun** [comment]
ah yeah, it was fixed in the latest update for sure, i tested it while still using ubuntu. i'm back on windows now though.

fwiw i was on on ubuntu 22.04.

**Comment by erkoiv** [comment]
Still an issue on 20.04 due to the older core package.

**Comment by rimpy-custom** [comment]
Closing for now. Reopen if issue wasn't fixed with latest Linux build 1.2.6.28.
